content/download/index.html.haml

```diff
@@ -212,22 +212,49 @@ title: Jenkins download and deployment
             &nbsp;
           %span.icon-gear-menu-o/
 
+%hr
+  NOTE: Packages with the
+  %span.icon-gear-menu-o
+  gear icon are maintained by third parties.
+  Such packages may be not as frequently updated as packages supported by the Jenkins project directly.
+
 %h2
   Deploying Jenkins in public cloud
 
-%p
-  There are public cloud vendors providing their own Jenkins distributions on marketplaces.
-  Such distributions may be used to quickly deploy Jenkins and, in many cases,
+%p{:style=>'margin-bottom:1cm;'}
+  Many public cloud vendors provide their own Jenkins installation guides and packages.
+  The guides provide instructions to deploy, maintain, and upgrade on the specific public cloud.
+  Such guides may be used to quickly deploy Jenkins and, in many cases,
   to get an instance preconfigured to be used within the public cloud
   (e.g. bundled plugins, integrations with public cloud services, etc.).
 
-  %div
-    %a{third_party_href('https://aka.ms/jenkins-on-azure')}
```

Comment:
@xuzhang3 this intentional?

Reply:
@rtyler has Microsoft shared anything with you on their plans or decisions on that service?

Reply:
seems like it's gone, no mention of it in the docs anymore.